### ISSUE TITLE
[debian-chroot] missing symlink?

### ISSUE THREAD

**Issue by mcondarelli** [SynoCommunity/spksrc#1512]
It seems to me there's (at least) a missing symlink:
> ln -s /proc/mounts /etc/mtab
Symptom is:
> root@syno0:/# df
> df: cannot read table of mounted file systems: No such file or directory

Also locales are a bit strange, but I didn't analyze it yet:
> syno0> /var/packages/debian-chroot/scripts/start-stop-status chroot
> bash: warning: setlocale: LC_ALL: cannot change locale (en_US.utf8)
> root@syno0:/#

This is in a freshly installed (DSM 5.1-5022 Update 2), running debian-chroot.
Did I miss something?

**Comment by Dr-Bean** [comment]
Yes, you missed the help page ;) 
https://github.com/SynoCommunity/spksrc/blob/develop/spk/debian-chroot/src/app/help/enu/index.html

Then, as far as the mtab goes: you can use `cp /proc/mounts /etc/mtab`, or not use it at all.

**Comment by mcondarelli** [comment]
AAAAARRRGGGHHH!!
You are right, first time through I followed instuctions, second I was careless.
The whole reason to use debian-chroot is I wanted to run some programs on my SynoStation (DS415+), in particular I wanted to setup a Redmine server, along with SVN & GIT integration.

I had some problems due to the "overblown" `/etc/apt/sources.list` making it problematic proper package upgrade, but those seem over, now (I would advise to stick to a single distribution, IMHO mixing wheezy, jessie, sid and experimental is calling for trouble; I kept jessie).
I also cured the "df" problem with a `ln -s /proc/mounts /etc/mtab` (I suggest to make this a standard).

Redmine is now working, along with chrooted sshd.
Now I need to fix Git & Svn.

Thanks for having made this possible with debian-chroot.
